selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -794,6 +794,19 @@ public void waitMarkerInPositionAndClick(MarkerLocator markerLocator, int positi
     seleniumWebDriverHelper.waitAndClick(By.xpath(format(markerLocator.get(), position)));
   }
 
+  /**
+   * Waits marker with specified {@code markerLocator} on the defined {@code position} and move
```

Comment:
`position` could confuse a little so as it's about the `line` number, not just about specific position in line.